docs/changehistory/NextVersion.md

```diff
@@ -1 +1,17 @@
 # NextVersion <!-- omit from toc -->
+
+Table of contents:
+
+- [@itwin/appui-react](#itwinappui-react)
+  - [Changes](#changes)
+  - [Fixes](#fixes)
+
+## @itwin/appui-react
+
+### Changes
+
+- Promoted `FrameworkToolAdmin` to _beta_. #618
+
+### Fixes
+
+- Correctly handle `WidgetState.Unloaded` and keep widget content unmounted when widget is unloaded. #617
```

Comment:
This has not been merged yet.

Reply:
Removed, I thought yours would go first, but you changed it now :)